packages/jaeger-ui/src/components/TracePage/index.tsx

```diff
@@ -430,15 +475,20 @@ export function mapStateToProps(state: ReduxState, ownProps: TOwnProps): TReduxP
     id,
     searchUrl,
     trace,
+    traceTimeline: state.traceTimeline,
   };
 }
 
 // export for tests
 export function mapDispatchToProps(dispatch: Dispatch<ReduxState>): TDispatchProps {
   const { fetchTrace } = bindActionCreators(jaegerApiActions, dispatch);
   const { archiveTrace, acknowledge: acknowledgeArchive } = bindActionCreators(archiveActions, dispatch);
-  const { focusUiFindMatches } = bindActionCreators(timelineActions, dispatch);
-  return { acknowledgeArchive, archiveTrace, fetchTrace, focusUiFindMatches };
+  return {
+    acknowledgeArchive,
+    archiveTrace,
+    fetchTrace,
+    ...((bindActionCreators(timelineActions, dispatch) as unknown) as TDispatchPropsTimelineViewer),
```

Comment:
Not sure if the preferred way to do this. Seen it at few places and is definitely shorter than destructuring all the functions and then assign them again there is more stuff passed to the component than necessary needed.

Reply:
I admittedly have a bias towards verbosity, but I find it clearer and preferable to destructure everything that is necessary and pass it along, rather than `(bindActionCreators(actions, dispatch) as unknown) as TType`.

It's a little more verbose in this case, but I'd still prefer it over the cast.